DynamicProgramming/FordFulkerson.java

```diff
@@ -3,46 +3,50 @@
 import java.util.LinkedList;
 import java.util.Queue;
 import java.util.Vector;
+import org.checkerframework.common.value.qual.*;
+import org.checkerframework.checker.index.qual.*;
 
 public class FordFulkerson {
     final static int INF = 987654321;
     // edges
-    static int V;
+    static @Positive int V;
     static int[][] capacity, flow;
 
     public static void main(String[] args) {
         System.out.println("V : 6");
         V = 6;
-        capacity = new int[V][V];
+        capacity = (int @MinLen(6) [] @MinLen(6) [])new int[V][V];
 
-        capacity[0][1] = 12;
-        capacity[0][3] = 13;
-        capacity[1][2] = 10;
-        capacity[2][3] = 13;
-        capacity[2][4] = 3;
-        capacity[2][5] = 15;
-        capacity[3][2] = 7;
-        capacity[3][4] = 15;
-        capacity[4][5] = 17;
+        ((int @MinLen(6) [] @MinLen(6) []) capacity)[0][1] = 12;
```

Comment:
You can avoid all of these by annotation the declaration of `capacity` at line 13 (you might need to split it from the declaration of `flow`)

Reply:
Yes, I changed the annotation at line 13. But, in general, I don't think we should have made change in line no 13. Instead, there should be a better way of writing the annotation it in the main method.